packages/xo-web/src/xo-app/settings/audit/index.js

```diff
@@ -371,6 +378,20 @@ export default decorate([
             {_('auditCheckIntegrity')}
           </ActionButton>
         </div>
+        {state.showDisabledRecordWarning && (
+          <Link
+            to={{
+              pathname: '/settings/plugins',
+              query: {
+                s: 'audit',
```

Comment:
Better query to avoid false positives:
```suggestion
                s: 'id%3A%2F^audit%24%2F',
```

Reply:
It doesn't work.